data/mods/blazemod/blaze_blob_parts.json

```diff
@@ -1054,12 +629,8 @@
     "damage_modifier": 10,
     "folded_volume": 1,
     "breaks_into": [ { "item": "glowiegrow" } ],
-    "requirements": {
-      "repair": {
-        "time": 30000,
-        "using": [ [ "blobrep", 1 ] ]
-      }
-    },
+    "description": "A living, glowing blob.  It consumes blob feed and produces electrical power, allowing it to be used as a reactor.  It also glows, and can be turned on to illuminate several squares inside the vehicle.  It also does not currently work.",
```

Comment:
`It also does not currently work.`
Does this refer to it being deactivated or refer to non-functionality due to engine not accepting bfeed yet?
If the latter, should probably put it no next line as a comment with "should work after fuel types corrected" or something.